av/video/frame.pyx

```diff
@@ -236,7 +236,7 @@ cdef class VideoFrame(Frame):
 
         import numpy as np
 
-        if frame.format.name == 'yuv420p':
+        if frame.format.name in ('yuv420p', 'yuvj420p'):
```

Comment:
What does the `j` mean that doesn't affect the shape?

Reply:
`yuvj420p` has the same shape as `yuv420p`. The difference in color ranges: 

- `yuv420p` according to rec 709 (https://en.wikipedia.org/wiki/Rec._709#Digital_representation) has a nominal range of [16..235] for Y channel and CB and CR channels have a nominal range of [16..240]. rec 601 has limited ranges also. It's main recommendations for `yuv420p` (https://en.wikipedia.org/wiki/YUV) 


- `yuvj420p` like I420, but with a full range ("digital", 0-255) (https://wiki.videolan.org/YUV#J420)